src/tidyversetopandas/tidyversetopandas.py

```diff
@@ -20,4 +20,5 @@ def arrange(df: pd.DataFrame, *col_name:str):
     >>> df = pd.DataFrame(np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]), columns=['a', 'b', 'c'])
     >>> df_sorted = tidyversetopandas.arrange(df, 'a', 'c')
     """
-    pass
\ No newline at end of file
+
+    return df.sort_values(by=list(col_name), ascending = ascending)
```

Comment:
nit: end in an extra line
```suggestion
    return df.sort_values(by=list(col_name), ascending = ascending)
    
```